students/ejackie/session02/series.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+
+"""
+a template for the series assignment
+"""
+
+
+def fibonacci(n):
+    """ compute the nth Fibonacci number """
+    if n == 0:
+        return 0
+    elif n == 1:
+        return 1
+    else:
+        return fibonacci(n-2) + fibonacci(n-1)
+
+
+def lucas(n):
+    """ compute the nth Lucas number """
+    if n == 0:
+        return 2
+    elif n == 1:
+        return 1
+    else:
+        return lucas(n-2) + lucas(n-1)
+
+
+def sum_series(n, n0=0, n1=1):
```

Comment:
This function was intended to be universally correct. Your implementation only works for fibonacci and lucas series.
E.g. it would not work for a series with n0=5 and n1=3
The idea here was to add one layer of abstraction.